jax/_src/lax/control_flow.py

```diff
@@ -2574,39 +2574,27 @@ def _cumred_dtype_rule(name, operand, *args, **kw):
                     "of number.".format(name, np.dtype(operand.dtype).name))
   return dtypes.canonicalize_dtype(operand.dtype)
 
-cumsum_p = lax.standard_primitive(
```

Comment:
This definition was confusing when I was trying to understand the CPU translation rules. I did not realize that the translation rule was overriden below in the file.